java-extensions/jni-connector/src/main/java/com/starrocks/jni/connector/ScannerHelper.java

```diff
@@ -39,6 +41,20 @@ public static ClassLoader createChildFirstClassLoader(List<File> preloadFiles, S
         return classLoader;
     }
 
+    public static ClassLoader createModuleClassLoader(String moduleName) {
+        String basePath = System.getenv("STARROCKS_HOME");
+        List<File> preloadFiles = new ArrayList<>();
+        preloadFiles.add(new File(basePath + "/lib/jni-packages/starrocks-hadoop-ext.jar"));
+        File dir = new File(basePath + "/lib/" + moduleName);
+        preloadFiles.addAll(Arrays.asList(Objects.requireNonNull(dir.listFiles())));
+        dir = new File(basePath + "/lib/common-runtime-lib");
+        preloadFiles.addAll(Arrays.asList(Objects.requireNonNull(dir.listFiles())));
+        dir = new File(basePath + "/lib/hadoop/common");
```

Comment:
add  jars in this dir actually fix this issue?

Reply:
yes, because in hadoop/common, there is azure.jar file.

So azure.jar and hadoop-fs.jar can be loaded by the same class loader.